modules/gapi/CMakeLists.txt

```diff
@@ -162,6 +162,9 @@ set(gapi_srcs
     # Python bridge
     src/backends/ie/bindings_ie.cpp
     src/backends/python/gpythonbackend.cpp
+
+    # Utils (ITT tracing)
+    src/utils/gapi_itt.cpp
```

Comment:
Don't think there is a need in `gapi` in the name of this file

Reply:
Fixed, thanks!